### PR TITLE
Replace way to set up eth0 in agw_install.sh

### PR DESCRIPTION
**Pull request by tmdzk**
Signed-off-by: Timothée Dzik <timdzik@fb.com>

## Summary

Replace way to set up eth0 in agw_install.sh

## Test Plan

sh agw_install.sh

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/deploy/agw_install.sh [hunk 1]
```diff
@@ -36,7 +36,8 @@ if [[ ! $INTERFACES == *'eth0'*  ]] || [[ ! $INTERFACES == *'eth1'* ]] || ! grep
   sed -i 's/GRUB_CMDLINE_LINUX=""/GRUB_CMDLINE_LINUX="net.ifnames=0 biosdevname=0"/g' /etc/default/grub
   # changing interface name
   grub-mkconfig -o /boot/grub/grub.cfg
-  sed -i 's/enp1s0/eth0/g' /etc/network/interfaces
```
Comment (xjtian) [inline lte/gateway/deploy/agw_install.sh:39]: should we keep this around and just make sure that it never fails? `sed || true` for example. or does the `auto eth0` handle this

Comment (tmdzk) [reply lte/gateway/deploy/agw_install.sh:39]: yes auto should handle this. 
The sed || true is a nice option but that one gives us more flexibility.

**Comment by tmdzk** [comment]
@xjtian @shanku9  can you guys review that ? we want to test it in the lab.

File: lte/gateway/deploy/agw_install.sh (hunk 1 above)
Comment (tmdzk) [reply lte/gateway/deploy/agw_install.sh:39]: auto will fix it. 
`Sed || true` is a good option but that one offered by @shanku9  gives us more flexibility in terms of conf

**Review by ardzoht — APPROVED** [review]

**Pull request merged**